LOGIC_GATES/LOOP_TOOLS/BREAK/BREAK.py

```diff
@@ -0,0 +1,36 @@
+from flojoy import flojoy, DataContainer, SmallMemory, NodeReference
+from typing import Optional, Any
+
+memory_key = "loop-info"
+
+
+@flojoy
+def BREAK(
+    referred_node: NodeReference,
+    default: Optional[DataContainer] = None,
+) -> DataContainer:
```

Comment:
Should this even return anything? I think it actually should just return `None` to prevent users from connecting anything to the output, a break node should always be a leaf in the DAG

Reply:
I agree this should return nothing!